src/plugins/vis_type_vega/public/data_model/vega_parser.ts

```diff
@@ -116,20 +161,20 @@ export class VegaParser {
     // eslint-disable-next-line import/namespace
     const logger = vega.logger(vega.Warn); // note: eslint has a false positive here
     logger.warn = this._onWarning.bind(this);
-    this.spec = vegaLite.compile(this.vlspec, logger).spec;
+    this.spec = vegaLite.compile(this.vlspec, { logger }).spec;
```

Comment:
This is changing the runtime behavior, was it done on purpose?

Reply:
No, I haven't done that on purpose. Somehow it changed.